### DISCUSSION TITLE
How can i use Custom Elements i blazor webView?

### DISCUSSION THREAD

**Discussion by sajjadarashhh** [dotnet/maui#20055] [Q&A]
Hi every body
i've been used Blazor Custom Elements in blazor wasm project that used custom elements in d3.js and d3 will be add that elements to DOM and blazor will rendered them
for now in maui i don't have RootComponents in builder.services
and i can use RegisterCustomElement in Maui Project
how can i configure that?

**Comment by Xyncgas** [comment]
instead of
```
builder.Services.AddRazorComponents()
    .AddInteractiveServerComponents(options =>
    {
        options.RootComponents.RegisterForJavaScript<Quote>(identifier: "quote",
          javaScriptInitializer: "initializeComponent");
    });
```
use
```
builder.RootComponents.RegisterForJavaScript<Quote>(identifier: "quote", 
    javaScriptInitializer: "initializeComponent");
```

**Reply by sajjadarashhh** [reply]
In Maui these not available
